README.md

```diff
@@ -12,14 +12,14 @@ The version of Cluster API Provider Cloud Director and Installation that are com
+|:---------------------------------------------------------------------------------:|:-------------------------:|:--------------------------------------------------------------------------------------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------:|:--------------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|  [main](https://github.com/vmware/cluster-api-provider-cloud-director/tree/main)  |           36.0+           | 10.3.1+ <br/>(10.3.1 needs hot-patch to prevent VCD cell crashes in multi-cell environments) |                                          [1.4.0](https://github.com/kubernetes-sigs/cluster-api/releases/tag/v1.4.0)                                           | <ul><li>1.25</li><ul><li>1.24</li><li>1.23</li><li>1.22</li><li>1.21</li><li>1.20</li></ul> | <ul><li>[1.4](https://github.com/vmware/cloud-director-named-disk-csi-driver/releases/tag/1.4)</li> <ul><li>[1.3.1](https://github.com/vmware/cloud-director-named-disk-csi-driver/releases/tag/1.3.1)</li><li>[1.3.0](https://github.com/vmware/cloud-director-named-disk-csi-driver/releases/tag/1.3.0)</li></ul> | <ul><li>[1.4.0](https://github.com/vmware/cloud-provider-for-cloud-director/releases/tag/1.4.0)</li><ul><li>[1.3.0](https://github.com/vmware/cloud-provider-for-cloud-director/releases/tag/1.3.0)</li><li>[1.2.0](https://github.com/vmware/cloud-provider-for-cloud-director/releases/tag/1.2.0)</li></ul> |
```

Comment:
There seems to be a formatting issue with the unordered list in the list of kubernetes versions, CPI versions and CSI versions for main branch